src/debug/dbg.ts

```diff
@@ -431,7 +431,7 @@ namespace Debug {
                     const above = lane > 0 ? connectors[column][lane - 1] : 0;
                     let connector = connectors[column][lane];
                     if (!connector) {

```

Comment:
I actually fixed this on the module branch other way, to add a `None` case to this instead, that way it's not `never`.

Reply:
(Also thankfully post-modules I ensure all projects get checked, so we should catch these.)

Reply:
Though, I think both aren't quite right anyhow given I think what this is checking is not 0, but `undefined` via out-of-bounds.